tests/test-cases/log-level-validation.yaml

```diff
@@ -14,8 +14,6 @@ tests:
     expect:
       diff: []
       exit_code: 1
-      stderr:
-        - "Error: Invalid log level 'XTrace'. Valid options are: \\[Trace Debug Info Warning Off\\]"
```

Comment:
Why did we remove this?

Reply:
Let me elaborate, golden snapshots can feel redundant, but the point here is that we can catch explicitly things we expect. My concern is if we rely solely on snapshots, we might not know what to look for.

Reply:
Okay got your point.
So you want to assert things that should exist for sure

Reply:
Just kind of a sanity check. Doesn't have to look for everything, but something about what's expected is nice.